roadmap.md

```diff
@@ -0,0 +1,61 @@
+# WebDriver BiDi roadmap
+
+## Real-world end-to-end user scenarios
+
+This document presents an overview of real-world end-to-end user scenarios we aim to enable via the WebDriver BiDi protocol. Each scenario requires one or more WebDriver BiDi commands and events to be specified, tested, and implemented across browser engines.
+
+### A. Extracting content
+
+This scenario loads a web page and uses script evaluation to extract content on the page (e.g. the headlines). In spec terms, this involves:
+
+- [x] [handling sessions](https://w3c.github.io/webdriver-bidi/#module-session)
+- [x] [creating a new `browsingContext`](https://w3c.github.io/webdriver-bidi/#command-browsingContext-create)
+- [x] [navigating to a URL](https://w3c.github.io/webdriver-bidi/#command-browsingContext-navigate)
+- [x] [evaluating JavaScript in the page context](https://w3c.github.io/webdriver-bidi/#command-script-evaluate)
+- [x] [serialization/deserialization of JavaScript values](https://w3c.github.io/webdriver-bidi/#data-types-protocolValue)
+- [ ] [sandboxed script execution](https://github.com/w3c/webdriver-bidi/issues/144)
+
+### B. Logging
+
+This scenario loads a web page and uses BiDi event subscription to efficiently get notified about each `console.log` entry. **This is not possible with classic WebDriver.** In spec terms, this involves:
+
+- [x] (some items from the previous milestone)
+- [x] [the `log` module](https://w3c.github.io/webdriver-bidi/#module-log)
```

Comment:
Are scenarios related to performance or network logs included in a different section?

Reply:
There is currently no scenario involving those types of logs. Feel free to propose one. We could potentially discuss this as a follow-up PR, once this one lands — WDYT?

Reply:
Makes sense. We can do that later.

Reply:
I'm currently working on improvements for this document and should have something later today. I think that those scenarios that we already did should be all present as well and that in an order we initially decided on.

Reply:
Given that network logging is a feature that we have kinda high on the priority list since the beginning we should include it. With my proposal that I can share soon I have an e2e scenario included.